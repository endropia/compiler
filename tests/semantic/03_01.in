type int = integer;
type int_ = int;
var
 	a: int;
 	b: int_;
begin end.