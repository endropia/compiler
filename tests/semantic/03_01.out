program : Unnamed program
   alias
      type: integer
      int
   alias
      type: int
      int_
   var: 
      a
      type: int
   var: 
      b
      type: int_
   stmts:
      empty
scope     name                          class               
------------------------------------------------------------
0         integer                       primitive type      
0         double                        primitive type      
0         boolean                       primitive type      
0         char                          primitive type      
0         string                        primitive type      
1         int                           alias               
1         int_                          alias               
1         a                             variable            
1         b                             variable            
